c/driver/common/utils.c

```diff
@@ -28,7 +28,7 @@
 static size_t kErrorBufferSize = 256;
 static char kErrorPrefix[] = "[SQLite] ";
 
-void ReleaseError(struct AdbcError* error) {
+static void ReleaseError(struct AdbcError* error) {
```

Comment:
Hmm, were these being multiply defined somehow?

Reply:
Don't think it's an immediate issue but since these aren't exposed in the header I think safer to keep them scoped to only usable within the TU